CrisisTextLine/Elmer
# Consolidate WiW instantiations and sprinkle in es6

#### What's this PR do?
Consolidate WiW instantiations and sprinkle in es6 const
#### Where should the reviewer start?
#### How should this be manually tested?
You can run the tasks manually to make sure they still work via:
node console ${task file name} ${exported function name}

You can also npm test and run selected jobs.
#### Any background context you want to provide?
#### What are the relevant tickets?
https://admin.crisistextline.org/jira/secure/RapidBoard.jspa?rapidView=5&view=detail&selectedIssue=INT-209
#### Questions:
